Taskfile.yml

```diff
@@ -730,9 +730,14 @@ tasks:
   docs-fmt:
     deps: [fmt-docs]
 
+  gen-cts:
+    cmds:
+      - ./bin/opendocdb-cts fmt --dir=./website/docs/guides/cts
+      - ./bin/opendocdb-cts convert --dir=./website/docs/guides/cts ./website/docs/guides
+
   docs:
     desc: "Format, lint and build documentation"
-    deps: [fmt-docs]
+    deps: [gen-cts, fmt-docs]
```

Comment:
Deps run in parallel, so there is a race between `gen-cts` generating files and `fmt-docs` formatting them.